aep/0144.yaml

```diff
@@ -0,0 +1,33 @@
+aliases:
+  UpdateArrayOperation:
+    description: An Update Array operation is a post request that updates an array with add or remove operation
+    targets:
+      - formats: ['oas2', 'oas3']
+        given:
+          - $.paths[?(@property.match(/:(add|remove)([^/]*)$/))]
+
+rules:
+  aep-144-update-array-operation-id:
+    description: The operation ID must begin with the word add or remove.
+    message: The operation ID does not conform to AEP-144.
+    severity: error
+    formats: ['oas2', 'oas3']
+    given: '#UpdateArrayOperation[*].operationId'
+    then:
+      function: pattern
+      functionOptions:
+        match: '^(add|remove)[A-Z].*$'
+
+  aep-144-http-method:
+    description: An Update Array operation must be POST.
+    severity: error
+    formats: ['oas2', 'oas3']
+    given: '#UpdateArrayOperation'
+    then:
+      function: schema
+      functionOptions:
+        schema:
+          type: object
+          properties:
+            post: {}
+          additionalProperties: false
```

Comment:
This has the same problem as I noted in #36 -- we can't use `additionalProperties: false`. This needs to be changed to `not: anyOf:` as I described in the other PR.